### PR TITLE
adds a tyr helmet module and antenna spinoff

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
adds a tyr helmet module, sprites by kuro
adds an antenna spinoff, sprites by M4Y or whatever it is in russian

## Why It's Good For The Game
tyr helomet module kills head meta, finally
spinoff is pretty cool, i think it's the best sprite in the codebase, not at all bad

## Changelog
:cl:
add: Added antenna spinoff, and tyr helmet module
/:cl:

### CODE REVIEW

**Comment by Kuro-ai** [comment]
yes this is the best addition in the history of additions,I endorse this pr

**Comment by BraveMole** [comment]
I am bravemole and I endorse this PR

**Review by TheBimmer231 — APPROVED** [review]

**Comment by lKiseki** [comment]
BRUHMMER HAS BEEN REPLACED BY A CAT, DON'T BELIEVE THIS, KILL THIS PR WITH HOLY FIRE, CAT EARS MUST REMAIN A RELIC OF THE PAST

**Comment by lovegreenstuff** [comment]
remove all armor from helmets
make armor cover head

**Comment by Losenis** [comment]
Speedmerge

File: code/modules/modular_armor/helmets.dm [hunk 1]
```diff
@@ -115,3 +115,24 @@
 /obj/item/helmet_module/antenna/toggle_module(mob/living/user, obj/item/clothing/head/modular/parent)
 	var/turf/location = get_turf(src)
 	user.show_message("<span class='warning'>The [src] beeps and states, \"Current location coordinates: LONGITUDE [location.x]. LATITUDE [location.y]. Area ID: [get_area(src)]\"</span>", EMOTE_AUDIBLE, "<span class='notice'>The [src] vibrates but you can not hear it!</span>")
+
+/obj/item/helmet_module/antenna/cat
+	name = "Cantenna helmet module"
+	desc = "Designed for mounting on a Jaeger Helmet. When attached, this module is able to provide quick readuts of the users coordinates. Made by the original company of the antenna module, desperate for cash. Maybe they aimed for a more cuter spin? Who knows."
+	icon_state = "antenna_cat_obj"
+	item_state = "antenna_cat"
```
Comment (SplinterGP) [inline code/modules/modular_armor/helmets.dm:123]: denied

**Review by SplinterGP — CHANGES_REQUESTED** [review]

File: code/modules/modular_armor/helmets.dm (hunk 1 above)
Comment (Kuro-ai) [reply code/modules/modular_armor/helmets.dm:123]: you can't deny progress splinter

**Review by SplinterGP — CHANGES_REQUESTED** [review]
Also unathomic

**Comment by Pariah919** [comment]
jaeger is going to die

**Pull request closed without merging**